vault/resource_pki_secret_backend_root_sign_intermediate_test.go

```diff
@@ -14,32 +15,149 @@ import (
 	"github.com/hashicorp/terraform-provider-vault/testutil"
 )
 
-func TestPkiSecretBackendRootSignIntermediate_basic(t *testing.T) {
+func TestPkiSecretBackendRootSignIntermediate_basic_default(t *testing.T) {
 	rootPath := "pki-root-" + strconv.Itoa(acctest.RandInt())
 	intermediatePath := "pki-intermediate-" + strconv.Itoa(acctest.RandInt())
+	format := "pem"
 
 	resource.Test(t, resource.TestCase{
 		Providers:    testProviders,
 		PreCheck:     func() { testutil.TestAccPreCheck(t) },
 		CheckDestroy: testPkiSecretBackendRootSignIntermediateDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testPkiSecretBackendRootSignIntermediateConfig_basic(rootPath, intermediatePath),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "backend", rootPath),
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "common_name", "test Intermediate CA"),
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "ou", "test"),
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "organization", "test"),
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "country", "test"),
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "locality", "test"),
-					resource.TestCheckResourceAttr("vault_pki_secret_backend_root_sign_intermediate.test", "province", "test"),
-					resource.TestCheckResourceAttrSet("vault_pki_secret_backend_root_sign_intermediate.test", "serial"),
-				),
+				Config: testPkiSecretBackendRootSignIntermediateConfig_basic(rootPath, intermediatePath, ""),
+				Check:  testCheckPKISecretRootSignIntermediate("vault_pki_secret_backend_root_sign_intermediate.test", rootPath, format),
 			},
 		},
 	})
 }
 
+func TestPkiSecretBackendRootSignIntermediate_basic_pem(t *testing.T) {
+	path := "pki-root-" + strconv.Itoa(acctest.RandInt())
```

Comment:
nit: should we keep the variable names consistent for all 3 `basic` tests? Perhaps `rootPath` and `intermediatePath`

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/1330/commits/db3ac9f89a2d030d6de3aa9d6eeb8f976a480cb9